extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -329,7 +404,14 @@ int ConsumeWindowsEventLog::processQueue(const std::shared_ptr<core::ProcessSess
     session->putAttribute(flowFile, FlowAttributeKey(MIME_TYPE), "application/xml");
     session->getProvenanceReporter()->receive(flowFile, provenanceUri_, getUUIDStr(), "Consume windows event logs", 0);
     session->transfer(flowFile, Success);
-    session->commit();
+
+	flowFile = session->create();
+
+	session->write(flowFile, &WriteCallback(evt.rendered_text_));
+	session->putAttribute(flowFile, FlowAttributeKey(MIME_TYPE), "text/plain");
+	session->getProvenanceReporter()->receive(flowFile, provenanceUri_, getUUIDStr(), "Consume windows event logs", 0);
+	session->transfer(flowFile, Success);
+	session->commit();
 
     flowFileCount++;
```

Comment:
This is incorrect as we produce twice the amount, but that doesn't hurt as we ignore the return value where this function is called. So the question is: do we need it at all?

Reply:
i think that's vestigial from what @am-c-p-p originally produced. should be a good follow on task Alex.